### ISSUE TITLE
Query pending-packets-by-address should return err when more than 1 args are passed

### ISSUE THREAD

**Issue by anhductn2001** [dymensionxyz/dymension#1423]
when I test this command:
```
dymd query delayedack pending-packets-by-address rollappevm_1234-1 dym147dz3jasag6pqt764c2cwvms6ke5m5mtx8vrt7 --output json --home /var/cosmos-chain/dymension-hub-2c864b679ae9873d3bf795d9c40e58203fc2f9b766975a74c186adc8fa76ac78d --node tcp://dymension_100-1-val-0-TestIBCTransferSuccess_EVM:26657 --chain-id dymension_100-1
```
it doesn't return an error which makes me think that it can't find any pending packets

**Comment by anhductn2001** [comment]
Was it resolved in PR #1424? @mtsitrin

**Comment by mtsitrin** [comment]
yes
